packages/ui-server/routes/index.js

```diff
@@ -3,7 +3,9 @@ const router = express.Router();
 
 /* GET home page. */
 router.get("/", function(req, res, next) {
-	res.render("index", { title: "Express" });
+	res.json({ title: "Express" });
 });
 
+router.use(require('./api'));
```

Comment:
use `/api` as the base route for the server calls.
We would be using `/api/init` for json calls to render `/init` and same for other endpoints.
```suggestion
router.use('/api',require('./api'));
```